.github/scripts/status_update_common.py

```
@@ -0,0 +1,212 @@
+            "projectNo": REPO_CONFIG["project_no"],
+            "cursor": cursor
+          }
+          data = self.send_graphql_request(query, variables)
+          items = data.get("organization", {}).get("projectV2", {}).get("items", {})
+          item_nodes = items.get("nodes", [])
+          has_next_page = items.get("pageInfo", {}).get("hasNextPage", False)
+          cursor = items.get("pageInfo", {}).get("endCursor", None)
+
+          for item in item_nodes:
+              content = item.get("content")
+              if content and content.get("number") == self.issue_number and content.get("repository", {}).get("name") == REPO_NAME:
```

Ruff (0.8.2)

Undefined name `REPO_NAME`

(F821)